### PR TITLE
fix doc and dashboard description about pending datapoints

### PR DESCRIPTION
**Pull request by Haleygo**
See [our playground](https://play-grafana.victoriametrics.com/d/oS7Bi_0Wz_vm/victoriametrics-cluster-vm?orgId=1&var-ds=P996FABE17B5F6D1E&var-job=All&var-job_insert=All&var-job_select=All&var-job_storage=All&var-instance=All) for reference.

### CODE REVIEW

**Review by tenmozes — APPROVED** [review]

File: docs/VictoriaLogs/README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Set the `-retentionPeriod` to some big value (e.g. `100y` - 100 years) if logs s
 For example:
 
 ```sh
-/path/to/victoria-logs -retention.maxDiskSpaceUsageBytes=10TiB -retention=100y
+/path/to/victoria-logs -retention.maxDiskSpaceUsageBytes=10TiB -retentionPeriod=100y
```
Comment (hagen1778) [inline docs/VictoriaLogs/README.md:120]: this change is unrelated. Could you please remove it?

Comment (Haleygo) [reply docs/VictoriaLogs/README.md]: removed.
